src/main/java/sc/fiji/colourDeconvolution/StainMatrixIJ2.java

```diff
@@ -289,20 +286,19 @@ static double logify(int colourValue) {
     }
 
     private ImgPlus<UnsignedByteType>[] initializeColorTables(ImgPlus<UnsignedByteType>[] outputImages) {
-        byte[] rLUT = new byte[256];
-        byte[] gLUT = new byte[256];
-        byte[] bLUT = new byte[256];
+        byte[][] rLUT = new byte[3][256];
+        byte[][] gLUT = new byte[3][256];
+        byte[][] bLUT = new byte[3][256];
```

Comment:
This feels a bit unnecessary. We don't really need to build up 3 different byte[]'s right? We can simply reuse the variables each time?

Reply:
I did this because the array is by reference so only the last array is used that is why the images where green.
I didn't know another way to get it working.